go.sum

```diff
@@ -3,8 +3,25 @@ github.com/ProtonMail/bcrypt v0.0.0-20210511135022-227b4adcab57 h1:pHA4K54ifoogV
 github.com/ProtonMail/bcrypt v0.0.0-20210511135022-227b4adcab57/go.mod h1:HecWFHognK8GfRDGnFQbW/LiV7A3MX3gZVs45vk5h8I=
 github.com/ProtonMail/go-crypto v0.0.0-20210428141323-04723f9f07d7 h1:YoJbenK9C67SkzkDfmQuVln04ygHj3vjZfd9FL+GmQQ=
 github.com/ProtonMail/go-crypto v0.0.0-20210428141323-04723f9f07d7/go.mod h1:z4/9nQmJSSwwds7ejkxaJwO37dru3geImFUdJlaLzQo=
-github.com/ProtonMail/go-mobile v0.0.0-20201014085805-7a2d68bf792f h1:u2i2ZBaZNzyJlIpyZWRJAoYMYqKObfJxvja3lr2olWw=
 github.com/ProtonMail/go-mobile v0.0.0-20201014085805-7a2d68bf792f/go.mod h1:skQtrUTUwhdJvXM/2KKJzY8pDgNr9I/FOMqDVRPBUS4=
+github.com/cronokirby/safenum v0.11.0 h1:XD9ikQk+SuCwQ4LuYllHCQd2/LBUIEx/2QH3pckbMMo=
+github.com/cronokirby/safenum v0.11.0/go.mod h1:AWp82xwEqKcnrpJPXPa1m0gF/OY8dzgL17ubUBnVygA=
+github.com/cronokirby/safenum v0.13.0 h1:1TAQXC30E9RWmjKwU7LgywsMRJkA6UgL+vZaU6NJegQ=
+github.com/cronokirby/safenum v0.13.0/go.mod h1:AWp82xwEqKcnrpJPXPa1m0gF/OY8dzgL17ubUBnVygA=
+github.com/cronokirby/safenum v0.14.0 h1:etZ9FnkWojRAphX2PeOquGX8Um6NqE02fHwLi9Xswx0=
+github.com/cronokirby/safenum v0.14.0/go.mod h1:AWp82xwEqKcnrpJPXPa1m0gF/OY8dzgL17ubUBnVygA=
+github.com/cronokirby/safenum v0.15.0 h1:WnAWW4lpQkPSQIXvyUZe99iaR5IBGPA/eQmCdU2EEN4=
+github.com/cronokirby/safenum v0.15.0/go.mod h1:AWp82xwEqKcnrpJPXPa1m0gF/OY8dzgL17ubUBnVygA=
+github.com/cronokirby/safenum v0.25.0 h1:3owHW181y3Ns+7nMRH0W9JuLi1H63imy6EngBlh9VHQ=
+github.com/cronokirby/safenum v0.25.0/go.mod h1:AWp82xwEqKcnrpJPXPa1m0gF/OY8dzgL17ubUBnVygA=
```

Comment:
This is very nitpicky, and idk if there's a command for this, but maybe let's clean this up a bit
```suggestion
```

Reply:
`go mod tidy` did it, thanks